XmlFormatter/src/Settings/Adapter/PropertyAdapter.cs

```diff
@@ -126,5 +126,39 @@ public ISettingScope GetSubScope(string name)
         {
             return null;
         }
+
+        /// <summary>
+        /// This method is not supported by this adapter
+        /// </summary>
+        /// <param name="name">The name of the scope to remove</param>
+        /// <returns>Always true</returns>
+        public bool RemoveSubScope(string name)
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// Method is not supported by the adapter
+        /// </summary>
+        public void ClearSubScopes()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053278)